organizing_hub/tasks.py

```diff
@@ -115,7 +117,19 @@ def sync_contact_list_with_bsd_constituent(
     is_subscribed = False
     email_address = cons_email.findtext('email')
     is_subscribed = cons_email.findtext('is_subscribed') == '1'
-    if email_address is None or not is_subscribed:
+
+    """Check if member of sendable cons group if enabled"""
+    if EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ENABLED:
```

Comment:
I don't think this logic really belongs here? There is a filter for cons group: https://cshift.cp.bsd.net/page/api/doc#-----------------Filter-Specification-------------

I suggest we just filter by cons group on the initial query to BSD api, right? I don't see the point of doing it here so late in the process when we could do it up front.

Reply:
@ericbroder This is if we decide to not use a sendable cons group in the future, i.e. if it is no longer maintained or used by our email program, perhaps because we cull our list and just don't keep non-sendable addresses. So we'd just disable here and be able to send form our full list. If this isn't here we can only send to members of a particular cons group.

Reply:
Oh wait I misread your comment, one sec.

Reply:
Damn I didn't even see that filter, you're right, will change.

Reply:
@cjmabry I don't understand, you can make it an optional param on the api call and only call it when enabled. This doesn't seem to address my point that this is in the wrong location? Why would we query all constituents first, store them in db, and then check if they are sendable later before we send, when we could just (optionally) query it all up front and not have to handle those non-sendable constituents at all?

Reply:
@ericbroder Apparently BSD Python API doesn't support cons_group filter https://github.com/bluestatedigital/bsd-api-python/blob/master/bsdapi/Filters.py